api/shape/Arc.js

```diff
@@ -160,26 +161,27 @@ const ArcOp =
         break;
     }
     const [c1, c2] = await buildCorners(x, y, z)(null);
-    const result = reify({ c1, c2, start, end, sides, zag });
-    return result;
+    const result = await reify({ c1, c2, start, end, sides, zag });
+    const geometry = await result.toGeometry();
+    return geometry;
   };
 
-export const Arc = Shape.registerMethod2(
+export const Arc = Shape.registerMethod3(
```

Comment:
Three! 😃

Reply:
Yeah, once everything is up to 3, I think it'll be time to remove the numbers. :)